dipy/denoise/patch2self.py

```diff
@@ -315,8 +317,18 @@ def patch2self(data, bvals, patch_radius=[0, 0, 0], model='ridge',
         denoised_arr[:, :, :, idx[0]] = np.squeeze(denoised_dwi[..., i])
 
     # shift intensities per volume to handle for negative intensities
-    for i in range(0, denoised_arr.shape[3]):
-        shift = np.min(data[..., i]) - np.min(denoised_arr[..., i])
-        denoised_arr[..., i] = denoised_arr[..., i] + shift
+    if shift_intensity == True and clip_negative_vals == False:
```

Comment:
in python, for a boolean value, you should avoid `==True` and `is True`. Can you update that everywhere?

In this specific case, `if shift_intensity and not clip_negative_vals:`

For your information, a quick test:
```python
In [1]: 1 is True                                                                                                                                                                                            
Out[1]: False

In [2]: 1 == True                                                                                                                                                                                            
Out[2]: True
```